pkgs/development/libraries/elfio/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "elfio";
+  version = "3.8";
+
+  src = fetchFromGitHub {
+    owner = "serge1";
+    repo = "ELFIO";
+    rev = "Release_${version}";
+    sha256 = "1h6q4nf19wpgzrz0wycl60s200r642jv41k1ffmqncgf92qy4yyv";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
```suggestion
```

Please never overwrite phases. This breaks many of the hooks used inside nixpkgs.

Reply:
Okay. Removed it. Should I add `dontBuild` or `dontConfigure` then?

Reply:
Yes, please.